lib/src/supabase.dart

```diff
@@ -108,12 +203,9 @@ class Supabase {
     _initialized = true;
   }
 
+  /// The [LocalStorage] instance used to persist the user session.
   LocalStorage get localStorage => _localStorage;
 
-  Future<bool> get hasAccessToken => _localStorage.hasAccessToken();
-
-  Future<String?> get accessToken => _localStorage.accessToken();
-
```

Comment:
These methods were removed because they were not being used

Reply:
We should keep these helper methods. At least i used them in the flutter demo app 😄

Reply:
Why?

Reply:
you can use `localStorage.accessToken`

Reply:
true, but they are just more convenient.

Reply:
FYI at the start, we didn't have localStorage class. We used `hasAccessToken` and `accessToken` methods. After introducing localStorage class, i think it's better to keep these getter methods to prevent breaking change.

Reply:
I'd rather remove it now instead of removing it in a future version

Reply:
but I'll add them back

Reply:
@phamhieu done!